apps/prairielearn/src/pages/studentAssessments/studentAssessments.ts

```diff
@@ -3,11 +3,21 @@ import asyncHandler from 'express-async-handler';
 
 import { loadSqlEquiv, queryRows } from '@prairielearn/postgres';
 
+import logPageView from '../../middlewares/logPageView.js';
+
 import { StudentAssessments, StudentAssessmentsRowSchema } from './studentAssessments.html.js';
 
 const sql = loadSqlEquiv(import.meta.url);
 const router = Router();
 
+router.use([
+  function (req, res, next) {
+    res.locals.navSubPage = 'assessments';
+    next();
+  },
```

Comment:
Nit: you can probably move this directly into the template. See for example:

https://github.com/PrairieLearn/PrairieLearn/blob/93ea6a7176bb4bbf89a7be3f636c176d81cb4101/apps/prairielearn/src/ee/pages/administratorInstitutionSso/administratorInstitutionSso.html.ts#L34

Reply:
Moved this directly into the Navbar template in [15771e5](https://github.com/PrairieLearn/PrairieLearn/pull/10906/commits/15771e55e6dd9050a1d39214df8424edc073a18f).